homeassistant/components/luxtronik/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""The Luxtronik heatpump integration."""
+# region Imports
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_COORDINATOR, DOMAIN, PLATFORMS
+from .coordinator import LuxtronikCoordinator
+
+# endregion Imports
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Luxtronik from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    # Create API instance
+    coordinator = LuxtronikCoordinator.connect(hass, entry)
+
+    await coordinator.async_config_entry_first_refresh()
+    entry.async_on_unload(entry.add_update_listener(update_listener))
+
+    data = hass.data.setdefault(DOMAIN, {})
+    data[entry.entry_id] = {}
+    data[entry.entry_id][CONF_COORDINATOR] = coordinator
```

Comment:
```suggestion
    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
```
Can set the coordinator directly there as you probably aren't going to store anything else